autogen.sh

```diff
@@ -2,8 +2,16 @@
 
 autoreconf --install --symlink -f
 
+SYSTEM=`uname -s`
+
 libdir() {
-        echo $(cd $1/$(gcc -print-multi-os-directory); pwd)
+        if [ $SYSTEM = "Linux" ] ; then
+                echo $(cd $1/$(gcc -print-multi-os-directory); pwd)
+        elif [ $SYSTEM = "Darwin" ] ; then
+                echo "/usr/lib"
```

Comment:
I don't think this is correct.  If there is no /usr/lib64 it should default to /usr/lib.